administration/src/application/components/ApplyController.tsx

```diff
@@ -26,6 +27,10 @@ const ApplyController = () => {
     applicationStorageKey,
     lastCommitForApplicationForm
   )
+  const { loading: loadingPolicy, data: policyData } = useGetDataPolicyQuery({
```

Comment:
We need some error handling (ie. refetch button / error toast)... Maybe you can add a todo here. It's okay if we defer this until we have implemented the regionId query.

Reply:
added TODO